autoscheduler/frontend/src/tests/ui/dummyData.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Add above the `eslint-disable` saying like `// Disable so we can represent the data returned from api/sections/ with snake case` or whatever. Just wanna document the ignores generally, as there's been a few times that I've looked at old ones and am not positive why they're needed